### PR TITLE
Breadcrumb

### PR DESCRIPTION
**Pull request by florettewest**

### CODE REVIEW

File: components/breadcrumb/docs.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import { Playground, PropsTable } from 'docz';
+import { Breadcrumb } from './src/Breadcrumb';
+
+# Breadcrumb
+
+Write your docs here
+
+<Playground>
+	<Breadcrumb arrItem={[['Home', '/'], ['Personal', '/personal-banking/']]} />
```
Comment (jaortiz) [inline components/breadcrumb/docs.mdx]: Update docs for updated implementation

File: components/breadcrumb/examples/_utils.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/** @jsx jsx */
```
Comment (jaortiz) [inline components/breadcrumb/examples/_utils.js]: remove file as unused

File: components/breadcrumb/src/Breadcrumb.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+	const common = {
+		padding: breadcrumb.padding,
+		marginBottom: breadcrumb.marginBottom,
+		fontSize: breadcrumb.fontSize,
+		listStyle: 'none',
+	};
+
+	const childrenWithProps = Children.map(children, (child, index) =>
+		index === children.length - 1
```
Comment (jaortiz) [inline components/breadcrumb/src/Breadcrumb.js]: refactor to Children.count(children) as children is an opaque data structure and not always guaranteed to be an array

File: brands/BOM/tokens/index.js [hunk 4]
```diff
@@ -177,6 +177,18 @@ export default {
...
+		marginBottom: '21px',
+		fontSize: '13px',
+		color: COLORS.text,
+		separator: {
+			marginLeft: '3px',
+			marginRight: '3px',
+			color: COLORS.primary,
+		},
+	},
+
```
Comment (jonnystening) [inline brands/BOM/tokens/index.js]: use rems now :)

File: components/breadcrumb/src/Breadcrumb.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+	const common = {
+		padding: breadcrumb.padding,
+		marginBottom: breadcrumb.marginBottom,
+		fontSize: breadcrumb.fontSize,
+		listStyle: 'none',
+	};
+
+	const childrenWithProps = Children.map(children, (child, index) =>
+		index === Children.count(children) - 1 ? cloneElement(child, { last: 'true' }) : child
```
Comment (jonnystening) [inline components/breadcrumb/src/Breadcrumb.js]: true as a string? 🤔

File: components/breadcrumb/src/Crumb.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+			{last && <SrOnly>Current page:</SrOnly>}
+			{children}
+			{!last && (
+				<ArrowRightIcon
+					size="small"
+					color={breadcrumb.separator.color}
+					css={{
+						marginLeft: breadcrumb.separator.marginLeft,
+						marginRight: breadcrumb.separator.marginRight,
+					}}
```
Comment (jonnystening) [inline components/breadcrumb/src/Crumb.js]: Should probably add an `aria-hidden="true"` attribute on the icon

Comment (jonnystening) [reply components/breadcrumb/src/Crumb.js]: wondering if we should render these separator icons as background images like with , to better facilitate the non-react system 🤔

TM didn't do that for Tabcordion chevron icon, so maybe they have another solution up their sleeve?

@jaortiz ?

File: components/breadcrumb/docs.mdx [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+menu: Components
+route: /breadcrumb
+---
+
+import { Playground, PropsTable } from 'docz';
+import { Breadcrumb, Crumb } from './src';
+
+# Breadcrumb
+
+Write your docs here
```
Comment (jonnystening) [inline components/breadcrumb/docs.mdx]: Use `Breadcrumbs are styled navigational links used to indicate a user journey or path. They are a simple, effective and proven method to aid orientation.` for now

File: components/breadcrumb/docs.mdx [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+    		</Crumb>
+    		<Crumb>
+    			<a href="/personal-banking/">Personal</a>
+    		</Crumb>
+    		<Crumb>Credit cards</Crumb>
+    	</Breadcrumb>
+
+</Playground>
+
+<PropsTable of={Breadcrumb} />
```
Comment (jonnystening) [inline components/breadcrumb/docs.mdx]:
```suggestion
## Props

### Breadcrumb
<PropsTable of={Breadcrumb} />

### Crumb
<PropsTable of={Crumb} />
```

File: brands/BOM/tokens/index.js (hunk 4 above)
Comment (jonnystening) [reply brands/BOM/tokens/index.js]:
```suggestion
	breadcrumb: {
		padding: '6px 18px',
		marginBottom: '21px',
		fontSize: '13px',
		color: COLORS.text,
		separator: {
			marginLeft: '3px',
			marginRight: '3px',
			color: COLORS.primary,
		},
	},
```

File: brands/BOM/tokens/index.js [hunk 9]
```diff
@@ -177,6 +177,18 @@ export default {
 		borderWidth: '1px',
 	},
 
+	breadcrumb: {
```
Comment (jonnystening) [inline brands/BOM/tokens/index.js]:
```suggestion
	breadcrumb: {
		padding: '0.6rem 1.8rem',
		marginBottom: '2.1rem',
		fontSize: '1.3rem',
		color: COLORS.text,
		separator: {
			marginLeft: '0.3rem',
			marginRight: '0.3rem',
			color: COLORS.primary,
		},
	},
```

Comment (jonnystening) [inline brands/BOM/tokens/index.js]:
```suggestion
	breadcrumb: {
		padding: '0.6rem 1.8rem',
		marginBottom: '2.1rem',
		fontSize: '1.3rem',
		color: COLORS.text,
		separator: {
			marginLeft: '0.3rem',
			marginRight: '0.3rem',
			color: COLORS.primary,
		},
	},
```

File: components/breadcrumb/docs.mdx [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+    		<Crumb>
+    			<a href="/">Home</a>
+    		</Crumb>
+    		<Crumb>
+    			<a href="/personal-banking/">Personal</a>
+    		</Crumb>
+    		<Crumb>Credit cards</Crumb>
+    	</Breadcrumb>
+
+</Playground>
```
Comment (jonnystening) [inline components/breadcrumb/docs.mdx]: fix indentation

```suggestion
<Playground>
	<Breadcrumb>
		<Crumb>
			<a href="/">Home</a>
		</Crumb>
		<Crumb>
			<a href="/personal-banking/">Personal</a>
		</Crumb>
		<Crumb>Credit cards</Crumb>
	</Breadcrumb>
</Playground>
```

File: components/breadcrumb/docs.mdx [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+import { Playground, PropsTable } from 'docz';
+import { Breadcrumb, Crumb } from './src';
+
+# Breadcrumb
+
+Write your docs here
+
+<Playground>
```
Comment (jonnystening) [inline components/breadcrumb/docs.mdx]: fix indentation

```suggestion
<Playground>
	<Breadcrumb>
		<Crumb>
			<a href="/">Home</a>
		</Crumb>
		<Crumb>
			<a href="/personal-banking/">Personal</a>
		</Crumb>
		<Crumb>Credit cards</Crumb>
	</Breadcrumb>
</Playground>
```

File: components/breadcrumb/src/Crumb.js (hunk 6 above)
Comment (jaortiz) [reply components/breadcrumb/src/Crumb.js]: I think it's nicer to use the Icon component, going by the whole atomic reuse pattern we are aiming for. I think leave it as the Icon for now and put a note in the issue as to whether we should be doing this as a background image or not

Comment (jonnystening) [reply components/breadcrumb/src/Crumb.js]: We could still use the Icon component, consume its contents and render it out as a background image using the `svgToTinyDataURI` dependency.

Keep it how it is @florettewest. Will mark this for discussion.

Comment (florettewest) [reply components/breadcrumb/src/Crumb.js]: I updated this question in the issues for breadcrumb...

**Pull request merged**